visualization/income_edu/income.html

```diff
@@ -210,7 +209,7 @@
 			var chart_div = d3.select("#chart");
 			
 			// If this is a re-draw (e.g. window resize), first remove the existing chart
-			chart_div.select("svg").remove();	
+			chart_div.selectAll("*").remove();	
```

Comment:
- In general I'd have code like this in a separate JS file not in the HTML
- Probably decrease the size of the indent you're using. 2 spaces is generally fine, and makes it easier to read nested code